test/output_test.go

```diff
@@ -0,0 +1,77 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/golangci/golangci-lint/test/testshared"
+)
+
+func TestOutput_Stderr(t *testing.T) {
+	sourcePath := filepath.Join(testdataDir, "gci.go")
+	fmt.Println(filepath.Abs(sourcePath))
+
+	testshared.NewRunnerBuilder(t).
+		WithArgs(
+			"--disable-all",
+			"--print-issued-lines=false",
+			"--print-linter-name=false",
+			"--out-format=line-number,json:stderr",
+		).
+		WithDirectives(sourcePath).
+		WithTargetPath(sourcePath).
+		Runner().
+		Install().
+		Run().
+		ExpectHasIssue("testdata/gci.go:8: File is not `gci`-ed").
+		ExpectOutputContains(`"Issues":[`)
```

Comment:
the test already existed but I can improve it.

Reply:
Yeah I actually saw the other test after writing this which tbh actually made me more confused since I guess they could share some code and checks. If you think it's easy to have a shared way of doing it go for it but I don't have a super strong opinion on this

Reply:
I updated the tests